mava/wrappers/smac.py

```diff
@@ -281,7 +284,29 @@ def get_stats(self) -> Optional[Dict]:
         Returns:
             extra stats to be logged.
         """
-        return self._environment.get_stats()
+        stats = self._environment.get_stats()
+        stats["cumulative_win_rate"] = stats["win_rate"]
+        del stats["win_rate"]
+        return stats
+
+    def get_interval_stats(self) -> Optional[Dict]:
+        """Computes environment statistics which should be exclusively \
+            computed over a given number of evaluation episodes.
+
+        An example would be a win rate calculation where it is required to
+        keep track of the number of games won over a given set of evaluation
+        episodes.
+
+        Returns:
+           Win rate
+        """
+        extra_stats: Dict[str, Any] = {}
```

Comment:
Think it would make more sense to call this dict interval_stats

Reply:
probably evaluation_stats?

Reply:
but the get_stats method also returns evaluation stats

Reply:
Yeah you are right the win rate is related to an interval stats I will do that

Reply:
Done!